discovery-provider/alembic/versions/6a97af9e5058_route_id.py

```diff
@@ -0,0 +1,38 @@
+"""route_id
+
+Revision ID: 6a97af9e5058
+Revises: cf614359625e
+Create Date: 2019-11-14 17:07:02.579491
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+from src.utils import helpers
+
+# revision identifiers, used by Alembic.
+revision = '6a97af9e5058'
```

Comment:
@piazz might have to update this b/c of https://github.com/AudiusProject/audius-protocol/pull/176